app/components/buttons/play-button-main.jsx

```
@@ -1,41 +1,36 @@
-      }} {...props}>
-          <div className={twMerge("state-layer rounded-full relative flex items-center justify-center", className)}>
-              <md-ripple className={`${props.disabled ? "hidden" : ""}`}></md-ripple>
-              <div className="flex w-fit gap-3 material-symbols-outlined-filled">
-              {playing ? "pause" : "play_arrow"}
-              </div>
-          </div>
-      </button>
+  return (
+    <button 
+      className="play-btn" 
+      role='button' 
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)